### DISCUSSION TITLE
`git-revision-date-localized-plugin` Error while running `make serve`

### DISCUSSION THREAD

**Discussion by tomas-svoboda-ipf** [mkdocs/mkdocs#3848] [Q&A] [answered]
Hello community,

I would like to ask you for a hint how to solve following bug I am getting while trying to run `make serve`.

`ERROR   -  [git-revision-date-localized-plugin] Unable to read git logs of '/docs/docs'.  To ignore this error, set option 'fallback_to_build_date: true'`

I read 2 different Discussions and honestly, none of them is relevant as they are over a year old and using older registries (before you ask, YES I did try to do suggested workarounds from other Threads/Discussions).

I am using following registry: 9.5.40-insiders-4.53.14
My git version: git version 2.39.3 (Apple Git-146)

Yes, I tried to put `fallback_to_build_date: true` into `mkdocs.yml` and removed when Error was still here.

My entire `docker command under make serve: docker run -it --rm -u 501:20 --name mkdocs -p 8000:8000 -v /Users/user/git/docs:/docs registry.gitlab.com/company/documentation/docs:9.5.40-insiders-4.53.14`
By the way, this resulted into another Warning message: `The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested` but I solved this by `docker run --platform linux/amd64 -it --rm -u 501:20 --name mkdocs -p 8000:8000 -v /Users/user/git/docs:/docs registry.gitlab.com/company/documentation/docs:9.5.40-insiders-4.53.14`.

Anyway, `ERROR   -  [git-revision-date-localized-plugin] Unable to read git logs of '/docs/docs'.  To ignore this error, set option 'fallback_to_build_date: true'` remains.

Any advised more than appreciated.

**Comment by pawamoy** [comment, marked as answer]
Intuitively, I'd say that git-revision-date-localized-plugin needs the Git history to compute revision dates for each file, but that you build in a Docker image *without* the `.git` folder, so the plugin can't read the Git history. Not sure what's the recommendation here. You could try asking over at https://github.com/squidfunk/mkdocs-material since you seem to be using their Insiders project, or at https://github.com/timvink/mkdocs-git-revision-date-localized-plugin directly.
